cosmwasm/packages/wasmi-runtime/src/macros.rs

```diff
@@ -51,3 +51,27 @@ macro_rules! coalesce {
         wrapper()
     }};
 }
+
+#[macro_export]
+macro_rules! validate_const_ptr {
+    ($ptr:expr, $ptr_len:expr) => {{
+        if let Err(_e) = validate_const_ptr($ptr, $ptr_len) {
+            ::log::error!("Tried to access data outside enclave memory!");
+            return $crate::results::result_init_success_to_initresult(Err(
+                ::enclave_ffi_types::EnclaveError::FailedFunctionCall,
+            ));
+        }
+    }};
+}
+
+#[macro_export]
+macro_rules! validate_mut_ptr {
+    ($ptr:expr, $ptr_len:expr) => {{
+        if let Err(_e) = validate_mut_ptr($ptr, $ptr_len) {
```

Comment:
`validate_mut_ptr` should be fully qualified as `$crate::utils::validate_mut_ptr`